### PR TITLE
Bump up dependencies

### PR DESCRIPTION
**Pull request by 417-72KI**
Mainly, [Octokit.swift v0.11.0](https://github.com/nerdishbynature/octokit.swift/releases/tag/0.11.0) has been released.

### CODE REVIEW

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Error deserializing SwiftLint JSON response (): dataCorrupted(Swift.DecodingError.Context(codingPath: [], debugDescription: "The given data was not valid JSON.", underlyingError: Optional(Error Domain=NSCocoaErrorDomain Code=3840 "No value." UserInfo={NSDebugDescription=No value.})))
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 0adf373f2d840c29c4eacf7312b57c95af92cd5c
</p>

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Error deserializing SwiftLint JSON response (): dataCorrupted(Swift.DecodingError.Context(codingPath: [], debugDescription: "The given data was not valid JSON.", underlyingError: Optional(Error Domain=NSCocoaErrorDomain Code=3840 "No value." UserInfo={NSDebugDescription=No value.})))
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 0adf373f2d840c29c4eacf7312b57c95af92cd5c
</p>

File: Tests/DangerTests/DangerDSLTests.swift [hunk 1]
```diff
@@ -34,7 +34,7 @@ final class DangerDSLTests: XCTestCase {
         XCTAssertTrue(danger.runningOnGithub)
         XCTAssertTrue(danger.supportsSuggestions)
         XCTAssertNotNil(danger.git)
-        XCTAssert(danger.github.api.configuration.accessToken == "7bd263f8e4becaa3d29b25d534fe6d5f3b555ccf")
+        XCTAssert(danger.github.api.configuration.accessToken == "7bd263f8e4becaa3d29b25d534fe6d5f3b555ccf".data(using: .utf8)?.base64EncodedString())
```
Comment (danger-public) [inline Tests/DangerTests/DangerDSLTests.swift:37]: - :warning: Line should be 140 characters or less: currently 150 characters (`line_length`)

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **Tests/DangerTests/DangerDSLTests.swift#L48** - Line should be 140 characters or less: currently 150 characters (`line_length`)
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 1ec609b0537b305d3c6147a7042600673f88b73b
</p>

File: Tests/DangerTests/DangerDSLTests.swift [hunk 2]
```diff
@@ -45,7 +45,7 @@ final class DangerDSLTests: XCTestCase {
...
```
Comment (danger-public) [inline Tests/DangerTests/DangerDSLTests.swift:48]: - :warning: Line should be 140 characters or less: currently 150 characters (`line_length`)

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Error deserializing SwiftLint JSON response (): dataCorrupted(Swift.DecodingError.Context(codingPath: [], debugDescription: "The given data was not valid JSON.", underlyingError: Optional(Error Domain=NSCocoaErrorDomain Code=3840 "No value." UserInfo={NSDebugDescription=No value.})))
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 1ec609b0537b305d3c6147a7042600673f88b73b
</p>

File: Tests/DangerTests/DangerDSLTests.swift (hunk 1 above)
Comment (danger-public) [inline Tests/DangerTests/DangerDSLTests.swift:37]: - :warning: Line should be 140 characters or less: currently 150 characters (`line_length`)

File: Tests/DangerTests/DangerDSLTests.swift (hunk 2 above)
Comment (danger-public) [inline Tests/DangerTests/DangerDSLTests.swift:48]: - :warning: Line should be 140 characters or less: currently 150 characters (`line_length`)

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Error deserializing SwiftLint JSON response (): dataCorrupted(Swift.DecodingError.Context(codingPath: [], debugDescription: "The given data was not valid JSON.", underlyingError: Optional(Error Domain=NSCocoaErrorDomain Code=3840 "No value." UserInfo={NSDebugDescription=No value.})))
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 1ec609b0537b305d3c6147a7042600673f88b73b
</p>

File: Tests/DangerTests/DangerDSLTests.swift (hunk 1 above)
Comment (danger-public) [inline Tests/DangerTests/DangerDSLTests.swift:37]: - :warning: Line should be 140 characters or less: currently 150 characters (`line_length`)

**Comment by f-meloni** [comment]
Closing because https://github.com/danger/swift/pull/417 was merged

**Pull request closed without merging**